### PR TITLE
Updated plugins.md [Added UI plugins by Public Lab]

### PR DESCRIPTION
**Pull request by sagarpreet-chadha**
Hi all , 
We at Public Lab have made these 2 plugins which we are using in our [website](https://publiclab.org/people/) , we wish to share these plugins with the world 😄  : 

1. LEL : https://github.com/publiclab/leaflet-environmental-layers
2. LBLD : https://github.com/publiclab/leaflet-blurred-location-display 

Thank You :)

### CODE REVIEW

**Comment by sagarpreet-chadha** [comment]
@ghybs , @IvanSanchez , @mourner 
I hope this is the correct place to add to the lists . Kindly review , Thanks !

**Comment by jywarren** [comment]
I think it is - maybe we have to ask for a review? Or maybe folks are just busy. No worries, i'm sure folks will see it soon! Thanks @sagarpreet-chadha !

**Comment by sagarpreet-chadha** [comment]
@ghybs , @IvanSanchez , @mourner 
I hope this is the correct place to add to the lists . Kindly review , Thanks !

**Comment by sagarpreet-chadha** [comment]
@perliedman can you have a look? Thanks!

**Comment by ghybs** [comment]
Hi @sagarpreet-chadha,

Thank you for your contributiions!

Sorry for the delay before reviewing your PR... :confused: 

I agree your 2nd plugin "leaflet-blurred-location-display" (3rd from your organization) fits in the [UI plugin section](https://leafletjs.com/plugins.html#user-interface), like the 1st already submitted plugin from your organization (leaflet-blurred-location).

But for the 1st plugin of this PR (leaflet-environmental-layers), it seems to me that it would be better in the [data-providers](https://leafletjs.com/plugins.html#data-providers) section, since it uses predefined data sources.

Please feel free to correct me if I misunderstood the features of your plugins.

File: docs/plugins.md [hunk 1]
```diff
@@ -3798,6 +3798,30 @@ Buttons, sliders, toolbars, sidebars, and panels.
...
+			<a href="https://github.com/publiclab/leaflet-environmental-layers">leaflet-environmental-layers
+			</a>
+		</td>
+		<td>
+			Collection of different environmental map layers in an easy to use Leaflet library <a href="https://publiclab.github.io/leaflet-environmental-layers/example/index.html#3/43.00/-46.26/BL2">Demo</a>.
+		</td>
+		<td>
+			<a href="https://github.com/publiclab">Public Lab</a>
+		</td>
+	</tr>
```
Comment (ghybs) [inline docs/plugins.md]: Please could you move this plugin to the [data-providers](https://leafletjs.com/plugins.html#data-providers) section of the Plugins page?

**Comment by sagarpreet-chadha** [comment]
@ghybs done :)

**Pull request merged**